lib-compose/src/main/java/com/what3words/components/compose/maps/providers/mapbox/W3WMapBox.kt

```diff
@@ -82,6 +78,7 @@ fun W3WMapBox(
             .filterNotNull()
             .onEach { currentCameraState ->
                 mapView?.mapboxMap?.let { mapboxMap ->
+                    Log.d("XXX","cameraPositionState ${mapboxMap.cameraState.zoom}")
```

Comment:
we can remove the log if it is not necessary

Reply:
My bad. Thanks Quoc